File: FlightManagementSystem/src/main/java/com/cg/flight/app/dto/FlightDTO.java
package com.cg.flight.app.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
@Data
@AllArgsConstructor
@NoArgsConstructor
public class FlightDTO {
private int flightId;
private String carrierName;
private String sourceAirport;
private String destinationAirport;
private String duration;
}

File: src/app/flight-dto.ts
export class FlightDTO {
    flightId:number;
    carrierName:string;
    sourceAirport:string;
    destinationAirport:string;
    duration:number;

    constructor(
        flightId:number,
        carrierName:string,
        sourceAirport:string,
        destinationAirport:string,
        duration:number
    )
    {
        this.flightId=flightId;
        this.carrierName=carrierName;
        this.sourceAirport=sourceAirport;
        this.destinationAirport=destinationAirport;
        this.duration=duration;
    }  
}
